docs/apps/srt-live-transmit.md

```diff
@@ -183,6 +184,19 @@ instead of `IP_ADD_MEMBERSHIP` and the value is set to `imr_sourceaddr` field.
 Explanations for the symbols and terms used above can be found in POSIX
 manual pages, like `ip(7)` and on Microsoft docs pages under `IPPROTO_IP`.
 
+### Medium: RTP
+
+RTP over UDP is supported for both input and output.
```

Comment:
What exactly is the "RTP output" here?

All I can see is that the RTP is the same as UDP for output. This at least isn't RTP. You would have to construct the RTP header to have the RTP output.

Reply:
Yes, that is a fair point - there's no attempt to do that, it's just possible to use rtp:// as the scheme for the output URL and, as it says on the next line down this will behave identically to UDP.

Would it be preferable to you to have RTP as input only and fail with invalid_argument or similiar if someone were to attempt to use a RTP output URL, or just to make the documentation clearer?

Reply:
Made RTP input only via e6c0687c5aee924460cf5ee5c0acd3b4d9f05029